consensus/istanbul/backend/engine_test.go

```diff
@@ -76,9 +76,12 @@ func makeCommittedSeals(hash common.Hash) [][]byte {
 // block by one node. Otherwise, if n is larger than 1, we have to generate
 // other fake events to process Istanbul.
 func newBlockChain(n int, items ...interface{}) (*blockchain.BlockChain, *backend) {
+	// some tests use custom test account
+	var key *ecdsa.PrivateKey
+
 	// generate a genesis block
 	genesis := blockchain.DefaultGenesisBlock()
-	genesis.Config = params.TestChainConfig
+	genesis.Config = &params.ChainConfig{ChainID: big.NewInt(1)}
```

Comment:
Is this okay with other tests? Why `ChainID` shoud be 1?

Reply:
`ChainID` is also 1 at `params.TestChainConfig`. It's okay with other tests.